### DISCUSSION TITLE
Создание плейлиста из всех лайкнутых треков исполнителя

### DISCUSSION THREAD

**Discussion by AlexandrovRoman** [Chimildic/YaMuTools#55] [Идея] [answered]
Добавить такую возможность в кнопку дискография на странице исполнителя

**Comment by 3270303** [comment]
вс, 12 июн. 2022 г., 13:52 Alexandrov Roman ***@***.***>:

**Reply by kisoid** [reply]
1) заходишь на страницу исполнителя и нажимаешь Дискография -> Все треки. Переименовываешь получившийся плейлист в "Плейлист А"
2) заходишь на страницу исполнителя _снова_ и нажимаешь Дискография -> Все треки. Переименовываешь получившийся плейлист в "Плейлист Б"
3) заходишь в "Плейлист А" и нажимаешь Вычитание треков -> Выберите плейлисты, треки которых надо удалить из текущего: "Мне нравится"
4) заходишь в "Плейлист Б" и нажимаешь Вычитание треков -> Выберите плейлисты, треки которых надо удалить из текущего: "Плейлист А"

В "Плейлист Б" получаешь то, что тебе нужно.

**Reply by AlexandrovRoman** [reply]
Ок. Спасибо за способ. Но по моему это слишком заморочено, не?

**Reply by kisoid** [reply]
> Ок. Спасибо за способ. Но по моему это слишком заморочено, не?

Может и заморочено, но это комплексная операция, состоящая из нескольких атомарных.

Следует развивать в расширении именно атомарные операции, ИМХО, чтобы потом пользователи могли из них как из конструктора составить какие угодно комплексные...
Иначе разработчику придётся по каждому чиху делать новую функцию и будет в расширении 100500 функций, каждой из которых будет пользоваться 1-2 человека...

Решать, конечно, разработчику, может действительно такая функция будет пользоваться популярностью...

**Comment by Chimildic** [comment, marked as answer]
Плейлист -> YaMuTools -> Фильтр треков -> Управление диз/лайками -> Оставить только лайки

В версии 0.8.4 добавлен пункт "все лайки". Дубли не удаляются.

**Reply by LyokhaOhm** [reply]
Нужно актуализировать автоматически создаваемое описание такого плейлиста.
Сейчас по умолчанию подставляется шаблонная фраза: «Все треки исполнителя на момент создания плейлиста.»
Например, сделать так: «Все ваши лайкнутые треки исполнителя на момент создания плейлиста.»
